src/data_format/mod.rs

```diff
@@ -9,9 +9,10 @@ mod v0;
 
 pub(crate) mod utils;
 
-use crate::blocker::Blocker;
 use crate::cosmetic_filter_cache::CosmeticFilterCache;
 
+use crate::blocker::Blocker;
+
```

Comment:
nit but I don't understand the reason to move `Blocker` into a separate paragraph by itself (similar change also in `src/data_format/v0.rs`)